server/market/bookrouter.go

```diff
@@ -229,7 +229,6 @@ out:
 			}
 
 			// Prepare the book/unbook/epoch note.
-			seq := subs.nextSeq()
```

Comment:
Good catch.  I broke this with `newEpochAction`, but it didn't apply to `matchProofAction` either...